.ci/build-mac.sh

```diff
@@ -40,7 +40,30 @@ export CPPFLAGS="-I$BREW_X64_PATH/include -msse -msse2 -mcx16 -no-pie"
 export LIBRARY_PATH="$BREW_X64_PATH/lib"
 export LD_LIBRARY_PATH="$BREW_X64_PATH/lib"
 
-git submodule update --init --recursive --depth 1
+export LLVM_DIR
+LLVM_DIR="BREW_X64_PATH/opt/llvm@16"
+git submodule update --init ./3rdparty/asmjit 
```

Comment:
Why are we doing individual submodule sync? To avoid llvm?
This solution scales poorly from a maintenance pov. Every time a new submodule is updated it breaks.

Reply:
See https://stackoverflow.com/questions/52179463/how-to-exclude-a-specific-git-submodule-from-update

Reply:
You can try:

git submodule -q update --init --depth=1 --jobs=8 $(awk '/path/ && !/llvm/ && !/SPIRV/ { print $3 }' .gitmodules)